src/Whmcs/Provider.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
```

Comment:
strict types please :)

Reply:
Sorry, I don't understand which types we are talking about